actions.go

```diff
@@ -286,22 +309,17 @@ func (c *Action) Fatalf(msg string, args ...interface{}) {
 	osExit(1)
 }
 
-// Infof prints a info-level message. The arguments follow the standard Printf
```

Comment:
The `fmt` package doesn't automatically add newlines

Reply:
Also, this won't work on Windows correctly

Reply:
But who uses `fmt` package for logging?
The new behavior is consistent with `log.Printf` as well as `Debugf/Warningf/Errorf` that all add newlines. One can replace one of those functions with Infof and get the same result without adding or removing \n.

Does the rest of this package work on Windows? For example, this code has the same issue, I guess: https://github.com/sethvargo/go-githubactions/blob/91ea68e3184f7727ede5960780eadce1fd7fee53/actions.go#L125

Reply:
> Does the rest of this package work on Windows?

Ok, I guess it does – it is easier for me to see that now, with other PR merged and GitHub Actions approved :)